lua/telescope/builtin.lua

```diff
@@ -647,6 +667,10 @@ builtin.find_files = function(opts)
     return
   end
 
+  if search_dirs then
+    insert_search_dirs(find_command, search_dirs)
```

Comment:
Instead of doing this here, can we do this above? I think it will be much more obvious what we're searching for. Change the `find_command` declarations above for each of the different executables.

Reply:
I didn't understand exactly what you mean here. Do you want me to do the `insert_search_dirs` immediately after each `find_command` declaration in the if statement?